assets/javascripts/discourse/initializers/composer-save-qa-mode.es6

```diff
@@ -0,0 +1,20 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
```

Comment:
This file is not running for me locally. I think it's due to the extension, i.e. it should be ``.js.es6``?